spec.emu

```diff
@@ -73,20 +84,17 @@ contributors: Nicolò Ribaudo
         <emu-alg>
           1. If _P_ is a Symbol, then
             1. Return ! OrdinaryGet(_O_, _P_, _Receiver_).
-          1. <ins>Let _m_ be _O_.[[Module]].</ins>
-          1. <ins>If _m_ is not a Cyclic Module Record, then</ins>
-            1. <ins>Perform ? EvaluateSync(_m_).</ins>
-          1. <ins>Otherwise if _m_.[[Status]] is not ~evaluated~ and ReadyForSyncExecution(_m_) is *true*, then:</ins>
-            1. <ins>Perform ? EvaluateSync(_m_).</ins>
+          1. <ins>Perform ? EnsureDeferredNamespaceEvaluation(_O_).</ins>
```

Comment:
nit: Instead of always performing this, it might be nice to have the gate on `if _m_.[[Deferred]]` here, and an assert in the `EnsureDeferredNamespaceEvaluation` instead.